components/MentorRegistration/components/PersonDetails.js

```diff
@@ -83,16 +83,10 @@ function PersonDetails({
             htmlFor="mentorProfile"
             className="mentorFormButton theme-button-color"
           >
-            {formData.mentorImg.image.length > 0
+            {formData.image.length > 0
```

Comment:
sorry this can't be ok... as there is no image.lenth you can `null` instead. as image is not in base64 as of now